examples/cpp/dds/BasicConfigurationExample/BasicConfigurationPublisher.cpp

```diff
@@ -192,6 +192,7 @@ bool HelloWorldPublisher::init(
     if (transient)
     {
         wqos.durability().kind = TRANSIENT_LOCAL_DURABILITY_QOS;
+        wqos.reliability().kind = RELIABLE_RELIABILITY_QOS;
```

Comment:
I would not do this. But if set, the warning after parsing should be removed.

Reply:
The warning of TLS hops being greater than 255 can also be omitted now.

Reply:
fair enought